src/app/api/fetch_user/route.ts

```diff
@@ -0,0 +1,32 @@
+"use server"
+import axios from "axios";
+import { NextResponse } from "next/server";
+import { sha512 } from 'js-sha512';
+
+export async function POST(req: Request) {
+    try {
+        const contestId = await req.json()
+        const currentUnixTime = Math.floor(Date.now() / 1000);
+        const hashHex = sha512.create().update(`123456/contest.standings?apiKey=${process.env.API_KEY}&contestId=${contestId}&time=${currentUnixTime}#${process.env.API_SECRET}`).hex()
+        const res: any = await axios.post(`https://codeforces.com/api/contest.standings?contestId=${contestId}&apiKey=${process.env.API_KEY}&time=${currentUnixTime}&apiSig=123456${hashHex}`)
+        const data = res.data.result.rows;
```

Comment:
use ?. instead of . 
Thats safer and you can also give fallback value, if no response is registered.

Reply:
Done